tests/post_training/data/wc_reference_data.yaml

```diff
@@ -3,6 +3,10 @@ tinyllama_data_free_backend_OV:
 tinyllama_data_aware_backend_OV:
   metric_value: 0.83084
 tinyllama_data_aware_awq_backend_OV:
-  metric_value: 0.81237
+  metric_value: 0.84439
 tinyllama_data_aware_awq_stateful_backend_OV:
-  metric_value: 0.81237
\ No newline at end of file
+  metric_value: 0.84439
+tinyllama_data_aware_awq_scale_estimation_backend_OV:
+  metric_value: 0.78801
```

Comment:
Is it latest references? Scale estimation + AWQ doesn't work for tiny-llama?
If this model has a poor answer on questions, probably another model should be used for testing, e.g. `stabilityai/stablelm-2-zephyr-1_6b`

Reply:
Updated